test/e2e/dns.go

```diff
@@ -309,10 +318,13 @@ var _ = Describe("DNS", func() {
 			fmt.Sprintf("%s.%s.svc", headlessService.Name, f.Namespace.Name),
 			fmt.Sprintf("_http._tcp.%s.%s.svc", headlessService.Name, f.Namespace.Name),
 			fmt.Sprintf("_http._tcp.%s.%s.svc", regularService.Name, f.Namespace.Name),
+			fmt.Sprintf("%s.%s.%s.svc.cluster.local", dnsTestPodHostName, dnsTestServiceName, f.Namespace.Name),
```

Comment:
we do the 1-2-3-4 names as part of the ".pod" subdomain right ?
Its not a change that was introduced in this PR. 
Were you thinking of some other 1-2-3-4 name that should be part of this PR ?